torchx/lib/torchx/backend.ex

```diff
@@ -202,6 +202,22 @@ defmodule Torchx.Backend do
     NIF.sum(to_ref(t), axes, keep_axes) |> from_ref(out)
   end
 
+  @impl true
+  def argmax(%T{type: out_type} = out, %T{} = t, opts) do
+    axe = opts[:axis] || 0
```

Comment:
To be clear, not having an axis is not the same as setting the default axis to zero! Here is an example:

```
iex(4)> Nx.iota({2, 3}, axis: 1) |> Nx.argmax()
#Nx.Tensor<
  s64
  2
>
iex(5)> Nx.iota({2, 3}, axis: 0) |> Nx.argmax()
#Nx.Tensor<
  s64
  3
>
iex(6)> Nx.iota({2, 3}) |> Nx.argmax()         
#Nx.Tensor<
  s64
  5
>
```

PS: singular of axes is axis. :)